.github/workflows/scala.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        scala-version: [ 2.12.10, 2.13.7 ]
-        java-version: ['8', '11']
+        scala-version: [ 2.12.16, 2.13.8 ]
+        java-version: ['11']
```

Comment:
we need java 8 because of https://docs.databricks.com/release-notes/runtime/10.4.html